crates/ide-db/src/syntax_helpers/suggest_name.rs

```diff
@@ -29,7 +29,7 @@ const USELESS_NAME_PREFIXES: &[&str] = &["from_", "with_", "into_"];
 /// # Examples
 /// `Option<Name>` -> `Name`
 /// `Result<User, Error>` -> `User`
-const WRAPPER_TYPES: &[&str] = &["Box", "Option", "Result"];
+const WRAPPER_TYPES: &[&str] = &["Box", "Arc", "Rc", "Option", "Result"];
```

Comment:
I wonder if there is some heuristic we can reliably apply here 🤔

Reply:
`AsRef` would cover the first three, but..

Reply:
`AsRef` is probably too general, likewise for `Deref`. I don't think there are any useful traits for this right now, so a list might be the best we have